co2eq/purchase/__tests__/footprints.test.js

```diff
@@ -183,3 +183,33 @@ describe('the purchase categories are correctly defined', () => {
     });
   });
 });
+
+describe('coicop codes are correctly attributed', () => {
+  Object.values(ENTRY_BY_KEY).forEach((entry) => {
+    if (entry.coicopCode) {
+      test(`the parent of ${entry.displayName} has an adequate coicop code`, () => {
+        const parent = entry.parentKey ? ENTRY_BY_KEY[entry.parentKey] : null;
+        if (parent && parent.coicopCode) {
+          expect(entry.coicopCode).toContain(parent.coicopCode);
+        }
+      });
+    }
+  });
+});
+
+describe('the associated purchase categories are correclty attributed', () => {
```

Comment:
Super important contribution I have here! 😉 

```suggestion
describe('the associated purchase categories are correctly attributed', () => {
```